<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalle proveedor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	
	background-color: rgb(6, 191, 249);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_codigo">
       <property name="text">
        <string>Código</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_codigo">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_nombre">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_nombre"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_direccion">
       <property name="text">
        <string>Dirección</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_direccion"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_codigo_postal">
       <property name="text">
        <string>Código Postal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_codigo_postal"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_poblacion">
       <property name="text">
        <string>Población</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_poblacion"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_provincia">
       <property name="text">
        <string>Provincia</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_provincia"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_telefono">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_telefono"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lbl_email">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="le_email"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lbl_contacto">
       <property name="text">
        <string>Contacto</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="le_contacto"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Móvil</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="le_movil"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_actualizar">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_borrar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(224, 27, 36);</string>
       </property>
       <property name="text">
        <string>Borrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Materias Primas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableView" name="tv_materias_primas"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cerrar">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(22, 183, 40);</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
